packages/cli/src/commands/generate/dbAuth/templates/resetPassword.tsx.template

```diff
@@ -32,7 +32,7 @@ const ResetPasswordPage = ({ resetToken }) => {
       }
     }
     validateToken()
-  }, [])
+  }, []) // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Should we just include the two deps here instead? (`validateResetToken` and `resetToken`)

Reply:
That would be a good question to look up in Redwood's coding standards :wink:

Generally in our own projects, when we are sure that dependency values never change during the on-screen lifetime of the component (that applies to `validateResetToken` and `resetToken`, does it?), we're leaving them out and explicitly disabling them with an eslint. 

I'm agnostic to the whatever way forward, i just feel there should be some rules-of-engagement to ensure consistency throughout the whole code base on the long run.

Reply:
@jtoar OK?

Reply:
We don't have any explicit rules/standards for this. But implicitly I think we always try to include the dependencies. But if we ever do disable a rule (it does happen) we always make sure to explain why it's unavoidable.

In this case though, I think we should just include the dependencies. (And if that leads to downstream errors related to component updates after unmount we'd need to handle those too.)

Reply:
> But implicitly I think we always try to include the dependencies. But if we ever do disable a rule (it does happen) we always make sure to explain why it's unavoidable.

We should make something like that explicit in some kind of coding standards / rules of engagement document. Then  everybody can be on the same page and the organizational knowledge isn't hidden within their members but upfront in the clear.

Fixed now via e77a657d4